src/test/java/jenkins/plugins/http_request/HttpRequestStepTest.java

```diff
@@ -670,7 +677,7 @@ public void nonExistentProxyAuthFailsTheBuild() throws Exception {
         WorkflowJob proj = j.jenkins.createProject(WorkflowJob.class, "proj");
         proj.setDefinition(new CpsFlowDefinition(
                 "def response = httpRequest url:'"+baseURL()+"/proxyAuth',\n" +
-                        "    proxy: 'http://proxy.example.com:8080',\n" +
+                        "    httpProxy: 'http://proxy.example.com:8080',\n" +
```

Comment:
This test had been passing erroneously. Adding a proper assertion exposed this bug in the test, which is now fixed.